modules/magic_enum/0.9.3/source.json

```diff
@@ -0,0 +1,8 @@
+{
+    "url": "https://github.com/Neargye/magic_enum/archive/refs/tags/v0.9.3.tar.gz",
```

Comment:
The CI failure should be fixed by referencing a release archive here. Just downloading [the auto-generated "Source code (tar.gz)"](https://github.com/Neargye/magic_enum/archive/refs/tags/v0.9.3.tar.gz) and reuploading it as a release archive is good enough.

Reply:
I opened an issue with magic_enum to see if they're willing to publish a release archive https://github.com/Neargye/magic_enum/issues/288. Main reason I marked this as a draft :P